torchci/rockset/metrics/__sql/lf_rollover_percentage.sql

```diff
@@ -0,0 +1,172 @@
+            normalized_jobs j
+            CROSS JOIN migrated_jobs mj
+        WHERE
+            1 = 1
+            AND j.job_name = mj.job_name -- AND STRPOS(j.name, mj.job_clean) > 0
+    ),
+    success_stats AS (
+        SELECT
+            bucket,
+            count(*) as group_size,
+            job_name,
+            workflow_name,
+            label,
+            IF(SUBSTR(label, 1, 3) = 'lf.', True, False) as lf_fleet,
+            SUM(
+                CASE
+                    WHEN conclusion = 'success' THEN 1
+                    ELSE 0
+                END
+            ) * 100 / (COUNT_IF(conclusion != 'cancelled') + 1) as success_rate,
+            -- plus one is to handle divide by zero errors
+            SUM(
+                CASE
+                    WHEN conclusion = 'failure' THEN 1
+                    ELSE 0
+                END
+            ) * 100 / (COUNT_IF(conclusion != 'cancelled') + 1) as failure_rate,
+            SUM(
+                CASE
+                    WHEN conclusion = 'cancelled' THEN 1
+                    ELSE 0
+                END
+            ) * 100 / COUNT(*) as cancelled_rate,
+            -- cancelled rate is calculated over all jobs
+            SUM(
+                CASE
+                    WHEN conclusion = 'success' THEN 1
+                    ELSE 0
+                END
+            ) as success_count,
+            SUM(
+                CASE
+                    WHEN conclusion = 'failure' THEN 1
+                    ELSE 0
+                END
+            ) as failure_count,
+            SUM(
+                CASE
+                    WHEN conclusion = 'cancelled' THEN 1
+                    ELSE 0
+                END
+            ) as cancelled_count,
+            COUNT(*) as total_count,
+            SUM(
+                CASE
+                    WHEN conclusion = 'success' THEN duration_min
+                    ELSE 0
+                END
+            ) / COUNT(*) as success_avg_duration,
+            SUM(
+                CASE
+                    WHEN conclusion = 'failure' THEN duration_min
+                    ELSE 0
+                END
+            ) / COUNT(*) as failure_avg_duration,
+            SUM(
+                CASE
+                    WHEN conclusion = 'cancelled' THEN duration_min
+                    ELSE 0
+                END
+            ) / COUNT(*) as cancelled_avg_duration,
+        FROM
+            comparable_jobs
+        GROUP BY
+            bucket,
+            job_name,
+            workflow_name,
+            label
+    ),
+    comparison_stats AS (
+        SELECT
+            lf.bucket,
+            SUM(lf.group_size + m.group_size) as total_jobs,
+            SUM(m.group_size) as meta,
+            SUM(lf.group_size) as lf,
+            CAST(SUM(m.group_size) as FLOAT) / SUM(lf.group_size + m.group_size) * 100 as percentage_meta,
+            CAST(SUM(lf.group_size) as FLOAT) / SUM(lf.group_size + m.group_size) * 100 as percentage_lf
```

Comment:
to be able to render both `percentage_meta` and `percentage_lf` on the same chart, you'll need a single column to contain all the percentages, and another column to say if it's a meta or lf result (let's call cat `fleet`).  Then in the metrics page you'll set `groupByFieldName` to `fleet`.

That'll result in a nice side-by-side comparison, where you can see one fleet going up in usage as the other goes down

Reply:
I think by itself percentage_lf is sufficient especially at the moment. However, we can chagne this later